README.md

```diff
@@ -11,3 +11,58 @@ The data points available for this project are all going to be from GH Archive.
 updated hourly, and will provide ample data for a daily update. 
 
 The data will be delivered in a SQL Database, and updated daily. 
+<<<<<<< Updated upstream
+
+##Models 
+
+There are different types of models available to users that will display 
+different fact/dimensional information in an efficient manner. The one we 
+plan on using is a Kimball Model. 
+
+### Different Fact and Dimensions Models 
```

Comment:
This first version of the readme looks neat, good job! 🙌 

To further improve it, I'd suggest we make it less academic and more like a real product documentation. Wdyt?

I'd consider making this part of the readme that describes Kimball modeling much shorter (1-2 sentences max) to avoid it becoming a theoretical presentation. Instead, let's have a readme that helps new users get started with your data product.